### ISSUE TITLE
Visor mode

### ISSUE THREAD

**Issue by GoogleCodeExporter** [mintty/mintty#171]
```
i know that its maybe not within the scope of mintty, but i would love a
visor mode like http://visor.binaryage.com/.

maybe something like this already exists through an external tool?
```

Original issue reported on code.google.com by `harding....@googlemail.com` on 13 Mar 2010 at 5:16

**Comment by mintty** [comment]
If you mean (as seen in some referred screenshots) that the window can be positioned to the screen top and sizes fullwidth, this is now supported, e.g.:
```mintty -s maxwidth -p 0,0 -s 0,10 -B void```

If you mean that mintty should be started with a hotkey, that is outside the scope of mintty itself.

If you mean anything else, describe it clearly and don't just refer to some other software.

**Comment by mintty** [comment]
Released in 2.2.1.

**Comment by mintty** [comment]
Actually, a Windows desktop shortcut can be associated with a hotkey ("Shortcut key").

**Comment by mintty** [comment]
Released 3.4.0.
